packages/common/src/utils/filters.ts

```diff
@@ -629,6 +629,7 @@ export const deleteFilterRuleFromGroup = (
 export const getDashboardFilterRulesForTile = (
     tileUuid: string,
     rules: DashboardFilterRule[],
+    needsExplicitTileOverride: boolean = false, // If true, we don't apply the default tile targets to the filter rule'
```

Comment:
We don't want to have a match if the field Id is "orders_state" and there is a column with the same name in a SQL chart. SQL charts only have filters if explicitly defined in the filter chart tile override.